src/app/login/login.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css']
+})
+export class LoginComponent implements OnInit {
+
+  constructor() { }
+
```

Comment:
Cool, so this is where we'll put in the logic for the login request, which I'll tie up in the backend

Reply:
Ah. How do I make calls to the API/backend from Angular?